src/parcel.js

```diff
@@ -63,8 +65,8 @@ export default class Parcel extends React.Component {
       if (SingleSpaContext && SingleSpaContext.Consumer) {
         return (
           <SingleSpaContext.Consumer>
-            {({mountParcel}) => {
```

Comment:
this line of code actually has the potential for a bug -- it is possible for the SingleSpaContext.Consumer component to not have a Producer who actually provides the `mountParcel` function. And destructuring undefined throws an error. So I've changed it to be more defensive.